tox.ini

```diff
@@ -2,27 +2,23 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{27,34,35}-dj{18,110}-{postgres,mssql}-wt{112},
-    py{27,34,35,36}-dj111-{postgres,mssql}-wt{112},
+    py{34,35,36}-dj111-{postgres,mssql}-wt{2},
+    py{34,35,36}-dj2-{postgres}-wt{2},
     flake8
 
 [testenv]
 basepython =
-    py27: python2.7
     py34: python3.4
     py35: python3.5
     py36: python3.6
 install_command = pip install -e ".[test]" -U {opts} {packages}
 commands =
     py.test --cov=wagtailtrans --cov-report=xml tests/
 deps =
-    dj18: django>=1.8,<1.9
-    dj110: django>=1.10,<1.11
     dj111: django>=1.11,<2.0
-    dj18-mssql: django-pyodbc-azure==1.8.17.0
-    dj110-mssql: django-pyodbc-azure==1.10.4.0
     dj111-mssql: django-pyodbc-azure==1.11.0.0
-    wt112: wagtail>=1.12,<1.13
+    dj2: django>=2.0
+    wt2: wagtail>=2.0
```

Comment:
Please note that this won't install any ``rcX`` versions, only the final ``2.0`` version. Also it would be nice to test against wagtail master, but allowing failures.

Reply:
I know, as soon as there is an RC i'll change it. 
Good point on the test against the master branch. I added an environment